spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/it/TestEntityRepository.java

```diff
@@ -30,6 +30,12 @@
 	@Query("select * from  test_entities_ci where id = @id_val")
 	List<TestEntity> findEntitiesWithCustomQuery(@Param("id_val") String id);
 
+	@Query("select * from  test_entities_ci where id = @id_val")
```

Comment:
Can we add a test where 
```
@Query("select age from people where last_name = @name")
Integer findAgeByLastname(String lastName);
```
?

Reply:
@meltsufin Added this test case and added the annotation setting for counts/exists just like how it's done in cassandra. PTAL